sarahricker/thinkery
# Programs text align and Exhibit meta removal

Centered text in the Program blocks on the /programs page and removed the visible date from the /exhibit archive page

## JIRA Ticket
[THNK-23](https://wpengine.atlassian.net/projects/THNK/issues/THNK-23)

## Description

This change center aligns the text in the Program boxes on the /programs page as well as removes post meta specifically for the date of publishing on the individual exhibits from the /exhibit archive page.

## Impacted Areas in Application

List general components of the site that this PR will affect:

thinkery.wpengine.com/programs

`.program-buttons`

thinkery.wpengine.com/exhibit

`entry-meta`

## Deploy Notes

Notes regarding deployment the contained body of work.
If you need to change something in WordPress after deploying, make the necessary steps crystal clear.

## Steps to Test or Reproduce

1. pull down branch
2. Load up the thinkery.wpengine.com/exhibit page and you should notice that the date isn't displaying on the Exhibit boxes.
3. Load up the thinkery.wpengine.com/programs page and the text will be center aligned at all window sizes

## Random GIF

Include one random GIF to "pay" your reviewers in humor.

![Thanks for reviewing my code!](https://media.giphy.com/media/xUA7b2S7SxhM1cGdsQ/giphy.gif)
